src/atoms/Button/index.js

```diff
@@ -29,12 +30,13 @@ function Button( props ) {
     unflex && styles['unflex'],
   ];
 
-  if (href) {
+  if (href || linkAttrs) {
```

Comment:
@sunnymis Why don't you just expose and spread `attr` in regardless? We only want the `<a>` if it's a link, right?

Reply:
@drewdrewthis The idea was linkAttrs would be an object containing
```
{
  href: '',
  rel: '',
  target: ''
}
```
because href is an attribute as well. We wouldn't have to pass both href and linkAttr but just one object.